frontend/src/modules/activity/components/activity-timeline.vue

```diff
@@ -61,86 +63,102 @@
-            <app-activity-content
-              v-if="activity.title || activity.body"
-              class="text-sm bg-gray-50 rounded-lg p-4"
-              :activity="activity"
-              :show-more="true"
-            >
-              <template v-if="platformDetails(activity.platform)?.activityDisplay?.showContentDetails" #details>
-                <div v-if="activity.attributes">
-                  <app-activity-content-footer
-                    :source-id="activity.sourceId"
-                    :changes="activity.attributes.lines"
-                    changes-copy="line"
-                    :insertions="activity.attributes.insertions"
-                    :deletions="activity.attributes.deletions"
-                  />
-                </div>
-              </template>
+            <template #dot>
+              <span
+                class="btn btn--circle cursor-auto p-2 bg-gray-100 border border-gray-200"
+                :class="`btn--${activity.platform}`"
+              >
+                <img
+                  v-if="platformDetails(activity.platform)"
+                  :src="
+                    platformDetails(activity.platform).image
+                  "
+                  :alt="`${activity.platform}-icon`"
+                  class="w-4 h-4"
+                />
+                <i
+                  v-else
+                  class="ri-radar-line text-base text-gray-400"
+                />
+              </span>
+            </template>
+          </el-timeline-item>
+        </template>
 
-              <template #bottomLink>
-                <div v-if="activity.url" class="pt-6">
-                  <app-activity-link
-                    :activity="activity"
-                  />
-                </div>
-              </template>
-            </app-activity-content>
-          </div>
-          <template #dot>
-            <span
-              class="btn btn--circle cursor-auto p-2 bg-gray-100 border border-gray-200"
-              :class="`btn--${activity.platform}`"
-            >
-              <img
-                v-if="platformDetails(activity.platform)"
-                :src="
-                  platformDetails(activity.platform).image
-                "
-                :alt="`${activity.platform}-icon`"
-                class="w-4 h-4"
-              />
-              <i
-                v-else
-                class="ri-radar-line text-base text-gray-400"
-              />
-            </span>
-          </template>
-        </el-timeline-item>
+        <div
+          v-if="!activities.length && !loading"
+          class="flex items-center justify-center pt-6 pb-5"
+        >
+          <div
+            class="ri-list-check-2 text-3xl text-gray-300 mr-4 h-10 flex items-center"
+          />
+          <p
+            class="text-xs leading-5 text-center italic text-gray-400"
+          >
+            This member has no activities in {{ getPlatformDetails(platform)?.name || 'the platform' }}
```

Comment:
Could you update "member" to "contact"? 😄 
And maybe as a fallback, instead of having "the platform" show "custom platforms"?